test/bin/pyutils/build_bootc_images.py

```diff
@@ -134,6 +139,21 @@ def set_rpm_version_info_vars():
     YMINUS2_RELEASE_REPO = common.run_command_in_shell(f"source {SCRIPTDIR}/get_rel_version_repo.sh; get_ocp_repo_name_for_version {YMINUS2_MINOR_VERSION}")
     RHOCP_MINOR_Y2 = YMINUS2_MINOR_VERSION
 
+    # Update environment variables based on the RPM version global variables.
+    # These are used for templating container files and images.
+    rpmver_globals_vars = [
+        'SOURCE_VERSION', 'MINOR_VERSION', 'PREVIOUS_MINOR_VERSION',
+        'YMINUS2_MINOR_VERSION', 'FAKE_NEXT_MINOR_VERSION', 'SOURCE_VERSION_BASE',
+        'CURRENT_RELEASE_VERSION', 'CURRENT_RELEASE_REPO', 'PREVIOUS_RELEASE_VERSION',
+        'PREVIOUS_RELEASE_REPO', 'RHOCP_MINOR_Y', 'RHOCP_MINOR_Y1',
+        'RHOCP_MINOR_Y2', 'YMINUS2_RELEASE_VERSION', 'YMINUS2_RELEASE_REPO'
+    ]
+    for var in rpmver_globals_vars:
+        value = globals().get(var)
+        if value is None:
+            raise Exception(f"The '{var}' global variable does not exist")
+        os.environ[var] = str(value)
```

Comment:
TBH I'm not a fan of this solution. Wouldn't some python native templating be better fit than duplicating what we have in bash (what in my opinion is a bad approach) with some nice data structures?